### ISSUE TITLE
Snapshots: Fail gracefully when triggered on non-btrfs systems

### ISSUE THREAD

**Issue by JosephKiranBabu** [freedombox/Plinth#1147]
A user on the mailing list reported of an unknown error when they tried to install Snapper on a non-btrfs file system.
https://lists.alioth.debian.org/pipermail/freedombox-discuss/2017-November/008172.html

We should show a warning message in the UI like the following, when a user clicks the Install button on a non-btrfs file system.
_"Your file system doesn't seem to be btrfs. Snapper only work on btrfs file systems."_

**Comment by JoKeyser** [comment]
Good idea; I would go one step further and *prevent* the user from clicking, by greying out/disabling the button in the first place, and show the explanation already.
